spec/controllers/errors_controller_spec.rb

```diff
@@ -0,0 +1,69 @@
+require "rails_helper"
+
+describe ErrorsController do
+  describe "GET #show" do
+
+    before :all do
+      secrets = Rails.application.secrets
+      @secret_key_base = secrets.secret_key_base
+      @secret_machine_fqdn = secrets.machine_fqdn
+      @secret_encryption_private_key_path = secrets.encryption_private_key_path
+      @secret_portus_password = secrets.portus_password
+    end
```

Comment:
You can get rid of this block because you have the same code inside of `before :each`